app/src/main/java/ch/sheremet/katarina/cocktailspro/model/Ingredients.java

```diff
@@ -111,9 +114,10 @@ public void setBeverageId(String mBeverageId) {
         this.mBeverageId = mBeverageId;
     }
 
+    @NonNull
     @Override
     public String toString() {
-        return String.format("Ingredients{mId=%d, mIngredient='%s', mMeasure='%s', mBeverageId='%s'}",
+        return String.format(Locale.getDefault(), "Ingredients{mId=%d, mIngredient='%s', mMeasure='%s', mBeverageId='%s'}",
```

Comment:
Regarding doc: https://docs.oracle.com/javase/8/docs/api/java/util/Formatter.html Optional locale as the first argument can be used to get locale-specific formatting of numbers. In this case we have only strings that came from server. I assume they won't be formated. WDYT?

Reply:
Well, actually mId is a number, so it is recommended to use a locale :)

Reply:
Yep, but it is an id :)

Reply:
But, anyway, thanks for Pull Request. Great work!